examples/quick_start/scripts/preproc/README.md

```diff
@@ -0,0 +1,250 @@
+  - **Count encoding** - represents the count of a given categorical value across the whole dataset (e.g. count of user past interactions) 
+  - **Target encoding** - represents one statistic of a categorical column conditioned on a target column. One example would be computing the average of click binary target segmented by the item id categorical values, which represents its Click-Through Rate (CTR) or likelihood to be clicked by a random user. [*Target encoding*](https://nvidia-merlin.github.io/NVTabular/v23.02.00/generated/nvtabular.ops.TargetEncoding.html#nvtabular.ops.TargetEncoding) is a very powerful feature engineering technique, and has been a key for many of our [winning solutions](https://medium.com/rapids-ai/winning-solution-of-recsys2020-challenge-gpu-accelerated-feature-engineering-and-training-for-cd67c5a87b1f) for RecSys competitions.
+  
+
+**Temporal features**
+- Extracting temporal features from timestamps, like day of week, day, month, year, quarter, hour, period of the day, among others. 
+- Compute the "age" of the item or how long the user is active in the system, e.g. by subtracting the interaction timestamp by the timestamp when the user/item were seen for the first time.
+- Trending features might also be useful: for example, including continuous features that accumulates the user engagement in a specific category of product the last month, quarter, semester.
+
+**Geographic features**
+- You can treat Zip codes, cities, states, countries as categorical features
+- If latitude/longitude are available, you can also compute distances, e.g. the distance between a hotel (item) location and the user location / airport / touristic landmark.
+- You can also enrich adding features based on external geolocation data (e.g. from census or government).,
+
+### Data set splitting
+There are many approaches for splitting (`--dataset_split_strategy`) train and evaluation data:
+- **random** - Examples are randomly assigned to train and eval sets (according to a percentage, `--random_split_eval_perc`).
+- **random by user** - It is like random but stratified by user.Ensures that users have examples in both train and eval sets. This approach doesn't provide cold-start users on eval set.
+- **temporal** - Uses a reference timestamp (`dataset_split_temporal_timestamp`) to split train and eval sets. Typically this is the most realistic approach, as when deployed models will not have access to future information when performing predictions.
+
+## Command line arguments
+In this section we describe the command line arguments of the preprocessing script.
+
+The input and format can be CSV, TSV or Parquet, but the latter is recommended for being a columnar format which is faster to preprocess. Output preprocessing format is parquet format.
+
+> You can check how to [setup the environment](../../ranking/README.md) to run `preprocessing.py` script with Docker.
+
+### Inputs
+```
+  --data_path
```

Comment:
if one already split train and val sets at their end up front, in that case, data-path would be `train set` path right? and to avoid any further split, users should also set `--dataset_split_strategy` to None?. or it is default None? If it is default None, that's fine.

Reply:
Yes, the default `--dataset_split_strategy` is None. You the eval and test set were already split, you just need to provide them in `--eval_data_path` and `--test_data_path`